bridge/telegram/html.go

```diff
@@ -53,13 +53,16 @@ func (options *customHTML) ListItem(out *bytes.Buffer, text []byte, flags int) {
 }
 
 func makeHTML(input string) string {
```

Comment:
This is the only code change part of Matterbridge. The rest is just changes in `vendor/`.